conan/tools/system/package_manager.py

```diff
@@ -37,7 +37,7 @@ def get_default_tool(self):
                            "brew": ["Darwin"],
                            "pacman": ["arch", "manjaro", "msys2"],
                            "choco": ["Windows"],
-                           "zypper": ["opensuse", "sles"],
+                           "zypper": ["opensuse", "sles", "opensuse-tumbleweed"],
```

Comment:
adding `"opensuse-tumbleweed"` as the distro related to zypper should solve the issue

Reply:
https://github.com/conan-io/conan/pull/3368 mentions `opensuse-leap`, and I have no idea what are the names for `sles`. This is why I chose to keep the behaviour of conans/client/tools/oss.py

Reply:
ok, I see, we can do some mix of both, and also test `opensuse-leap` and make sure that in the future if one distro returns `opensuse-newversion` it will not break...